publish.sh

```diff
@@ -31,13 +23,13 @@ else
     export PYPI_PACKAGE_REPOSITORY="test"
 fi
 
-# export version
-VERSION_LAST_TAG=$(git describe --abbrev=0 --tags 2>/dev/null)
 
```

Comment:
Maybe leave a comment here specifying that the image is being pushed to Pypi as an official release.